tree/ntupleutil/v7/src/RNTupleInspector.cxx

```diff
@@ -228,42 +229,67 @@ const std::vector<ROOT::ENTupleColumnType> ROOT::Experimental::RNTupleInspector:
 void ROOT::Experimental::RNTupleInspector::PrintColumnTypeInfo(ENTupleInspectorPrintFormat format, std::ostream &output)
 {
    struct ColumnTypeInfo {
-      std::uint32_t count;
-      std::uint64_t nElems, compressedSize, uncompressedSize;
+      std::uint64_t nElems = 0, compressedSize = 0, uncompressedSize = 0;
+      std::uint64_t nPages = 0;
+      std::uint32_t count = 0;
 
       void operator+=(const RColumnInspector &colInfo)
       {
          this->count++;
          this->nElems += colInfo.GetNElements();
          this->compressedSize += colInfo.GetCompressedSize();
          this->uncompressedSize += colInfo.GetUncompressedSize();
+         this->nPages += colInfo.GetNPages();
+      }
+
+      // Helper method to calculate compression ratio
+      double GetCompressionRatio() const
+      {
+         if (uncompressedSize == 0)
+            return 1.0;
+         return static_cast<double>(compressedSize) / static_cast<double>(uncompressedSize);
       }
    };
 
-   std::map<ROOT::ENTupleColumnType, ColumnTypeInfo> colTypeInfo;
+   std::map<ENTupleColumnType, ColumnTypeInfo> colTypeInfo;
 
+   // Collect information for each column
    for (const auto &[colId, colInfo] : fColumnInfo) {
       colTypeInfo[colInfo.GetType()] += colInfo;
    }
 
    switch (format) {
-   case ENTupleInspectorPrintFormat::kTable:
-      output << " column type    | count   | # elements      | compressed bytes  | uncompressed bytes\n"
-             << "----------------|---------|-----------------|-------------------|--------------------" << std::endl;
+   case ENTupleInspectorPrintFormat::kTable: {
```

Comment:
The added curly braces here and in the other `case`s seem unnecessary. Could you remove them?